src/python/WMCore/Services/Dashboard/ProcInfo.py

```diff
@@ -32,17 +32,14 @@
 """
 from __future__ import print_function
 from __future__ import division
-from builtins import object
-from builtins import str
-from past.utils import old_div
 from WMCore.Services.Dashboard.Logger import Logger
 import socket
 import os
 import re
 import time
 
 
-class ProcInfo(object):
+class ProcInfo:
```

Comment:
If you want you can still new class inheriting object without importing builtins object. But it is up to you.